home/signals.py

```diff
@@ -4,12 +4,17 @@
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 
-from stats.models import GuildMembership, RoleMembership
+
+ADMIN_PERMISSION_BITFLAGS = 0x8
```

Comment:
You should add leading zeros that pertain to the width of the flag. I think permission flags are 64-bit, which is admittedly quite long, but you could add enough to at least pad it to some multiple of 4 or 8.

Reply:
Done